src/hb-ot-var-mvar-table.hh

```diff
@@ -86,6 +86,9 @@ struct MVAR
     return (this+varStore).get_delta (record->varIdx, coords, coord_count);
   }
 
+  inline float apply_var (hb_tag_t tag, int coord) const
+  { return coord + (version.major ? get_var (tag, &coord, 1) : 0.f); }
```

Comment:
Does this look correct?

Reply:
Don't know.  Does it need version check?  I'd guess not.